### ISSUE TITLE
Full Video Tutorial - by me

### ISSUE THREAD

**Issue by savageduckerino** [mikealmond/hq-trivia-assistant#105]
https://www.youtube.com/watch?v=HTeVbMRELK0

**Comment by perfectusr** [comment]
I personally think we should keep this within the github community before it blows up

**Comment by savageduckerino** [comment]
Nope. public, someone is going to make a tutorial at one point so I might as well now.

**Comment by Tpixelminer** [comment]
I agree with @perfectusr, that video will blow up pretty quickly and thats not necessarily a good thing.

**Comment by redraskal** [comment]
I'm assuming they will eventually (if not already) only allow one connection at a time to the socket with the same user id / token, although you could just get another number and retrieve the token and not use HQ on that number. I heard they send the actual question information before it displays, so that will most likely be delayed a bit more to make the bots slower. Either way, bots are bots and you cannot really fix this issue. If they somehow block people connecting to HQ while not on a phone, you could use https://cloud.google.com/vision/docs/ocr and fetch the text from the screen and simply run the searches from the OCR results. (Not detectable at all :D)
